scripts/tasks/2_annotate_bad_segments_and_auto_bad_channels.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
<p>For matplotlib, I recommend 2 settings which might make your life easier when saving figures: save them in SVG with a transparent background.</p><p>Also a plot (picture) is a derivative, and the file should be locked. Basically, any file that is saved by the step is a derivative file and should be locked.</p><p> </p><pre class="ql-syntax" spellcheck="false">fname = derivatives_folder_plots_p / fname_stem / (fname_stem + "_0_bridges.svg")
fig.savefig(fname, transparent=True)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mscheltienne/eeg-flow/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
ok thanks, made the change. 
note that I left the savefig on the bridges cell rather that at the end where all derivatives are saved, not sure whether it should be left there or not!

Reply:
Yes that's fine, it doesn't need to be saved at the very end ;)